docker/proxy-nginx.Dockerfile

```diff
@@ -15,3 +17,7 @@ RUN  grep -P '^([A-Z_]*_PORT[A-Z_]*)\s*=\s*([0-9]){4,5}$' /root/address.py \
      | while IFS=$' \t=' read VAR VALUE; do \
          sed -i "s/{$VAR}/$VALUE/g" /etc/nginx/nginx.conf;\
      done
+
+RUN a=`getent hosts host.docker.internal | awk '{ print $1 }'` \
```

Comment:
Nitpicking, but in general `$()` is preferred over backticks for command substitution (see [BashFAQ/082](http://mywiki.wooledge.org/BashFAQ/082) for some more context).